CMakeLists.txt

```diff
@@ -30,6 +30,21 @@ option(BUILD_VC       "Build Vc library from source")
 
 include(CompilerSetup)
 
+EXECUTE_PROCESS (
+		COMMAND ${CMAKE_C_COMPILER} -print-file-name=libc.so.6
+		OUTPUT_VARIABLE GLIBC
+		OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+GET_FILENAME_COMPONENT (GLIBC ${GLIBC} REALPATH)
+GET_FILENAME_COMPONENT (GLIBC_VERSION ${GLIBC} NAME)
+STRING (REPLACE "libc-" "" GLIBC_VERSION ${GLIBC_VERSION})
+STRING (REPLACE ".so" "" GLIBC_VERSION ${GLIBC_VERSION})
+
+if(NOT (GLIBC_VERSION LESS 2.22))
+  SET(GCC_LINK_FLAGS "-lmvec -lm -ftree-vectorize -ffast-math")
```

Comment:
@amadio @xvallspl The `-ffast-math` causes the `backend` and `math` tests to fail. Could it be due to the [precision difference](https://stackoverflow.com/a/22135559) ? Any hints how to address this problem ?

Reply:
Strange. The tests don't seem to fail here.